lib/MetaCPAN/Model/Archive.pm

```diff
@@ -77,7 +77,11 @@ has _tempdir => (
     init_arg => undef,
     lazy     => 1,
     default  => sub {
-        return File::Temp->newdir;
+
+        my $scratch_disk = '/mnt/scratch_disk';
```

Comment:
We are starting to be better about hard-coded paths.  In this case, I think the path will be the same on the Vagrant box, but this will probably explode under Travis, so we at least need to special case that.